README.md

```diff
@@ -1,5 +1,7 @@
 # Licit
-Legal Aid 
+Legal Aid
+
+[![CircleCI](https://circleci.com/gh/CoWare/Licit/tree/develop.svg?style=svg)](https://circleci.com/gh/CoWare/Licit/tree/develop)
```

Comment:
- [ ] Add this repo to coveralls and not that of @bolorundurowb.

- [ ] We want the badge to show on all branches and not master alone